packages/app/src/api.ts

```diff
@@ -220,41 +221,62 @@ export function initializeApp(
  *
  * @param options - `Firebase.AppOptions` to configure the app's services, or a
  *   a `FirebaseApp` instance which contains the `AppOptions` within.
- * @param config - `FirebaseServerApp` configuration.
+ * @param config - Optional `FirebaseServerApp` settings.
  *
  * @returns The initialized `FirebaseServerApp`.
  *
  * @public
  */
 export function initializeServerApp(
   options: FirebaseOptions | FirebaseApp,
-  config: FirebaseServerAppSettings
+  config?: FirebaseServerAppSettings
 ): FirebaseServerApp;
 
+/**
+ * Creates and initializes a {@link @firebase/app#FirebaseServerApp} instance.
+ *
+ * @param config - Optional `FirebaseServerApp` configuration.
+ *
+ * @returns The initialized `FirebaseServerApp`.
+ *
+ * @public
+ */
```

Comment:
I think it's useful to add `@throws` tag on functions that throw, but the other functions in app don't do this, so maybe it'd just make it inconsistent which may be worse- feel free to ignore.

```
@throws {AppError.INVALID_SERVER_APP_ENVIRONMENT} - ... 
```